### ISSUE TITLE
Ruffle seems to not load at all for some sites (not CORS-related)

### ISSUE THREAD

**Issue by asdfugil** [ruffle-rs/ruffle#6412]
### Describe the bug

Ruffle seems to not load at all for some sites - resulting in `Adobe Flash Player is no longer supported`

Console message: `Error with Permissions-Policy header: Unrecognized feature: 'interest-cohort'.`

To reproduce:
Launch chromium: `chromium --user-data-dir=/tmp --disable-web-security` and go to https://web.archive.org/web/20021125121723/http://www.eyezmaze.com/ (You may need to reload it a few times to get an error)

*In this case, the issue is behind another issue concerning CORS requests, so web security needs to be disabled to reveal this issue.* (#4914)

### Expected behavior

Ruffle loads, and plays the content.

### Affected platform

Browser's extension

### Operating system

Debian 11 (Linux 5.15.0-0.bpo.3-amd64)

### Browser

Chromium 99.0.4844.51 built on Debian 11.2, running on Debian 11.2

### Additional information
Even more examples: https://web.archive.org/web/20100417211419/http://www.eyezmaze.com/eyezblog_en/blog/2009/06/grow_ver3_remake.html#monster
https://web.archive.org/web/20100418061758/http://www.eyezmaze.com/

Screenshot:
![image](https://user-images.githubusercontent.com/42699250/156988884-af21c15c-72f7-43ce-9459-a99b5072d620.png)

**Comment by n0samu** [comment]
Chromium-based browsers do not wait for extensions to finish loading before starting to load a webpage. Ruffle cannot insert itself into a webpage until it has checked its own configuration to make sure it's supposed to be running. (See https://github.com/ruffle-rs/ruffle/blob/master/web/packages/extension/src/content.ts#L109)

Wayback Machine has a script file `/_static/js/bundle-playback.js` that interferes with Ruffle, only if it is loaded first. So basically, if a Wayback Machine page loads _too quickly_, its playback.js script loads before the Ruffle extension does, and you get the "Flash Player is not supported" message instead of the Ruffle embed.

For Wayback Machine specifically, you can avoid this issue by adding the `id_` suffix to your URLs like this: https://web.archive.org/web/20021125121723id_/http://www.eyezmaze.com/. The `id_` suffix loads the original saved webpage instead of one that includes the playback.js script. But there are similar issues on other websites that aren't as easily solvable.

Side note: I'm not getting any CORS error on the page - I'm not sure why you might be. I was able to confirm your issue without using any special launch parameters.

**Comment by n0samu** [comment]
Thanks to @p0008874 we finally have a solution for this issue! See the FAQ: https://github.com/ruffle-rs/ruffle/wiki/Frequently-Asked-Questions-For-Users#wayback
